gvsbuild/projects/pkgconf.py

```diff
@@ -27,10 +27,9 @@ def __init__(self):
             self,
             "pkg-config",
             prj_dir="pkgconf",
-            archive_url="https://distfiles.dereferenced.org/pkgconf/pkgconf-1.8.0.tar.gz",
-            hash="d7b6fdb522d81c11f5a0e0a0629a9f5480809ec90e595058674c1517822dfb8c",
+            archive_url="https://distfiles.dereferenced.org/pkgconf/pkgconf-1.9.3.tar.gz",
```

Comment:
I'm going to revert this change, pkgconf 1.9.x is a testing version in preparation for version 2.0, we should still with 1.8.x until it is finalized.